src/EPR.Calculator.Frontend.UnitTests/LocalAuthorityDataUtilTests.cs

```diff
@@ -0,0 +1,153 @@
+        }
+
+        [TestMethod]
+        public void GetLocalAuthorityData_MultipleItemsSameCountry_GroupsByCountry()
+        {
+            // Arrange
+            var multipleItems = new List<LocalAuthorityDisposalCost>
+        {
+            new LocalAuthorityDisposalCost
+            {
+                Id = 1, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc),
+                CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1,
+                LapcapDataTemplateMasterUniqueRef = "ENG-AL", Country = "England", Material = "Aluminium", TotalCost = 2210.45m
+            },
+            new LocalAuthorityDisposalCost
+            {
+                Id = 2, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc),
+                CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1,
+                LapcapDataTemplateMasterUniqueRef = "ENG-FC", Country = "England", Material = "Fibre composite", TotalCost = 2210m
+            },
+        };
+
+            // Act
+            var result = LocalAuthorityDataUtil.GetLocalAuthorityData(multipleItems);
+
+            // Assert
+            Assert.IsNotNull(result);
+            Assert.AreEqual(2, result.Count);
+            Assert.AreEqual("England", result[0].Country);
+            Assert.AreEqual("England", result[1].Country);
+        }
+
+        [TestMethod]
+        public void GetLocalAuthorityData_ItemWithSpecificMaterialType_HandlesCorrectly()
+        {
+            // Arrange
+            var items = new List<LocalAuthorityDisposalCost>
+        {
+            new LocalAuthorityDisposalCost
+            {
+                Id = 1, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc),
+                CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1,
+                LapcapDataTemplateMasterUniqueRef = "ENG-AL", Country = "England", Material = "Aluminium", TotalCost = 2210.45m
+            },
+            new LocalAuthorityDisposalCost
+            {
+                Id = 2, ParameterYear = "2024-25", EffectiveFrom = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc),
+                CreatedBy = "Test User", CreatedAt = new DateTime(2024, 8, 28, 10, 12, 30, DateTimeKind.Utc), LapcapDataMasterId = 1,
+                LapcapDataTemplateMasterUniqueRef = "ENG-FC", Country = "England", Material = "Other", TotalCost = 2210m
+            },
+        };
+
+            // Act
+            var result = LocalAuthorityDataUtil.GetLocalAuthorityData(items);
+
+            // Assert
+            Assert.IsNotNull(result);
+            Assert.AreEqual(2, result.Count);
+            Assert.AreEqual(MaterialTypes.Material, result[1].Material);
+        }
```

Comment:
Can you assert all the properties you are sending to the UI

Reply:
sure